topics/geneontology/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: GO, geneontology, gene-ontology
```

Comment:
Unfortunately this is an alias that will have many conflicts. One being github.com/topics/go. Also, `geneontology` is already the topic name so it doesn't need to be included in this list.

```suggestion
aliases: gene-ontology
```